src/components/AnonymousChat/BaseChatItem.tsx

```diff
@@ -110,6 +110,7 @@ const BaseChatItem = ({item, index, type}: BaseChatItemComponentProps) => {
       username={handleUserName(item, selectedChannel)}
       type={BaseChatItemTypeProps.ANON_CHAT}
       chatItem={item}
+      chatType={type}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of `chatType` prop to the component is a good change. It allows for more flexibility in handling different types of chats. However, it's important to ensure that this new prop is properly validated and handled in the component where it's used.
```diff
+ chatType={type}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->